Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.yml

```
@@ -2631,6 +2654,149 @@ script:
     - contextPath: SentinelOne.Accounts.Name
       description: The account name.
       type: string
+  - arguments:
+    - name: account_ids
```

According to our conventions
```suggestion
    - name: account_id
```